src/content/development/code-review/_index.en.md

```diff
@@ -89,6 +89,20 @@ ready to merge (has [required reviews](#two-non-author-committer-approvals-requi
 other relevant code has changed, the new job results don't tell us anything new. Since changes are constantly being merged, it's likely
 another merge commit and set of jobs will be necessary right before merging anyway.
 
+### Mark work-in-progress PRs as drafts
+
+To clearly indicate a Pull Request is still under development, not yet ready for review, mark it as a draft. It is not necessary to modify
+PR summaries or commit messages ("WIP" or "DO NOT MERGE"). Keeping the same PR summary keeps email notifications threaded, and using the
```

Comment:
```suggestion
PR summaries or commit messages to indicate this (eg "WIP" or "DO NOT MERGE"). Keeping the same PR summary keeps email notifications threaded, and using the
```

Reply:
I had a version like this, but it seemed unnecessarily verbose. Also the "this" is hanging on to a broad idea from the first sentence, which is more difficult to read than necessary IMHO.

Reply:
I did part of this in the latest.